ol3-viewer/apply_patches.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
Out of curiosity would `sh` suffice or are we happy to assume `bash` everywhere? (E.g., pretty safe bet on Linux I'd think.)